MaintenanceApi.yaml

```diff
@@ -17,6 +17,8 @@ tags:
   
 - name: Maintenance Component
   description: Maintenance Component data model endpoints  
+- name: Maintenance (Unscheduled)
```

Comment:
This doesn't line up with the tags called out in the unscheduled maintenance endpoints